interpreter/src/test/scala/pl/touk/nussknacker/engine/spel/SpelExpressionSpec.scala

```diff
@@ -233,13 +247,25 @@ class SpelExpressionSpec extends FunSuite with Matchers {
     parse[String]("4 + ''") shouldBe 'valid
   }
 
```

Comment:
What do we do about it?

Reply:
Can you mark the problematic part again? I think something broke during one of rebases.

Reply:
the code is commented out....